corva/app/scheduled.py

```diff
@@ -11,8 +10,8 @@ class ScheduledApp(BaseApp):
     group_by_field = 'app_connection_id'
 
     @property
-    def event_loader(self) -> ScheduledLoader:
-        return ScheduledLoader()
+    def event_loader(self):
```

Comment:
What are the `event_loader` methods doing now?

Reply:
ScheduledApp class will not be used anymore. Its logic will be extracted into middlewares and file deleted in [task](https://corvaqa.atlassian.net/browse/DC-754). I've changed the method, so tests don't fail.